.gitsubprojects

```diff
@@ -1,5 +1,5 @@
 # -*- mode: cmake -*-
-git_subproject(Brion https://github.com/BlueBrain/Brion.git 015515f)
+git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git 87a9ac3)
+git_subproject(Brion https://github.com/BlueBrain/Brion.git d1f9c43)
```

Comment:
That is a good question.
After looking at it it seems justified to list it here explicitly, because of the following:
MONSTEER_DEPENDENT_LIBRARIES Lunchbox
If you don't list Lunchbox here then you get a non-explicit leaking dependency, provided by either Brion or zeq (in whichever order they come). It makes it also more painful to update the Lunchbox version for this project because you have to make the SHA1 update "bubble up" until here through Brion, even though Brion doesn't need the update in this specific case (only ZEQ).